### PR TITLE
Docs integration with sentry

### PR DESCRIPTION
**Pull request by joaqcid**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ X ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: docs/recipes/integration-with-sentry.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler(),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
```
Comment (profanis) [inline docs/recipes/integration-with-sentry.md:52]: do you think we can use the `provideAppInitializer` which is part of angular 19?
https://angular.dev/api/core/provideAppInitializer?tab=api

File: docs/recipes/integration-with-sentry.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],
+};
+```
+
+Now, let's go over how to log NGXS actions as breadcrumbs.
+Create a service `NgxsSentryBreadcrumbsService`. 
+This service uses NGXS `Actions` dependendecy to listen to all events and send them to Sentry using `Sentry.addBreadcrumb`.
```
Comment (profanis) [inline docs/recipes/integration-with-sentry.md:63]:
```suggestion
This service uses NGXS `Actions` dependency to listen to all events and send them to Sentry using `Sentry.addBreadcrumb`.
```

File: docs/recipes/integration-with-sentry.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            Sentry.addBreadcrumb({
+              category: 'NGXS',
+              message: `${actionType} ${ctx.status}`,
+              level: 'info',
+              type: 'info',
+              data: typeof ctx.action === 'string' ? { data: ctx.action } : ctx.action,
+            });
+          }
+        }),
+        takeUntil(this.#destroyed$)
```
Comment (profanis) [inline docs/recipes/integration-with-sentry.md:91]: do you think we can use the `https://angular.dev/api/core/rxjs-interop/takeUntilDestroyed`?

On the other hand, if this stream lives for as long as the application is up and running, do we need to destroy that 🤔?

Comment (arturovt) [reply docs/recipes/integration-with-sentry.md:91]: Nope, it's _completed_ by NGXS when the root injector is destroyed.

File: docs/recipes/integration-with-sentry.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+  constructor() {
+    this.#subscribeToActions();
+  }
+
+  #subscribeToActions() {
+    const excludeAction = [] //You can add the actions you want to exclude from your breadcrumbs here;
+    this.#actions
+      .pipe(
+        tap((ctx) => {
```
Comment (arturovt) [inline docs/recipes/integration-with-sentry.md:79]: The `tap` body can be moved to subscribe block as thus the operator would be removed.

File: docs/recipes/integration-with-sentry.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  ],
+};
+```
+
+Now, let's go over how to log NGXS actions as breadcrumbs.
+Create a service `NgxsSentryBreadcrumbsService`. 
+This service uses NGXS `Actions` dependendecy to listen to all events and send them to Sentry using `Sentry.addBreadcrumb`.
+
+```ts
+@Injectable()
```
Comment (arturovt) [inline docs/recipes/integration-with-sentry.md:66]:
```suggestion
@Injectable({providedIn:root})
```

File: docs/recipes/integration-with-sentry.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+{
+  provide: NgxsSentryBreadcrumbsService,
+  useClass: NgxsSentryBreadcrumbsService,
+},
+{
+  provide: APP_INITIALIZER,
+  useFactory: () => () => {},
+  deps: [NgxsSentryBreadcrumbsService],
+  multi: true,
+}
```
Comment (arturovt) [inline docs/recipes/integration-with-sentry.md:114]:
```suggestion
provideAppInitializer(() => inject(NgxsSentry...))
```

**Review by markwhitfeld — APPROVED** [review]

**Pull request merged**